app/models/beta/search/intercept_message.rb

```diff
@@ -0,0 +1,22 @@
+module Beta
+  module Search
+    class InterceptMessage
+      include ContentAddressableId
+
+      content_addressable_fields :intercept_message
+
+      attr_accessor :intercept_message, :term, :message
+
+      def self.build(search_query)
+        return unless search_query.eql?((I18n.t "#{search_query}.title"))
```

Comment:
Should we downcase the input search query?

Reply:
You're right, i missed that. We do need to do that. Updating it with a short commit